tests/main/nfs-support/task.yaml

```diff
@@ -41,6 +54,46 @@ restore: |
     systemctl reset-failed snapd.service
     systemctl start snapd.service
 
+    # Depending on OS do cleanup appropriate for the host.
+    case "$SPREAD_SYSTEM" in
+        ubuntu-14.04-*)
```

Comment:
Why is ubuntu-16.04, 18.04 etc missing here?

Reply:
Maybe a comment here for our future selfs that may have the same question?

Reply:
I'll add a comment but the basic answer is that the cleanup is split between two chunks (stop, unmount, start-if-needed). Not all systems need the changes, specifically Ubuntu 16.04 and beyond have NFS preinstalled and working and we don't need to stop it.

Reply:
Can you look again?